tests/DartStableTest.php

```diff
@@ -19,5 +19,8 @@ class DartStableTest extends Base
         ...Base::GENERAL_RESPONSES,
         ...Base::LARGE_FILE_RESPONSES,
         ...Base::EXCEPTION_RESPONSES,
+        ...Base::QUERY_HELPER_RESPONSES,
+        ...BASE::PERMISSION_HELPER_RESPONSES,
+        ...BASE::ID_HELPER_RESPONSES
```

Comment:
```suggestion
        ...Base::PERMISSION_HELPER_RESPONSES,
        ...Base::ID_HELPER_RESPONSES
```

Seems this is problematic in all the tests, suggesting here is not efficient, please update the rest as well